tests/deduplicate/test_deduplicate.py

```diff
@@ -1,4 +1,6 @@
 import pyspark
+import sys
+sys.path.append('../..')
 from homework.deduplicate.deduplicate import Deduplicator
```

Comment:
Could you please remove all that additions in tests? If you would like to run tests, please use 'python -m pytest tests/' in project root, or IDE